pom.xml

```
@@ -15,6 +15,11 @@
     <module>extensions</module>
     <module>docs</module>
   </modules>
+  <scm>
+    <connection>:git:git@github.com:quarkiverse/quarkus-groovy.git</connection>
```

```suggestion
    <connection>scm:git:git@github.com:quarkiverse/quarkus-groovy.git</connection>
```
I maybe wrong but I think that it should start with "scm", don't you agree?